container-registry/aws/ecr/versions.tf

```diff
@@ -1,13 +1,24 @@
+provider "skopeo2" {
+  destination {
+    login_username = data.aws_ecr_authorization_token.current.user_name
+    login_password = data.aws_ecr_authorization_token.current.password
+  }
+}
+
 terraform {
   required_version = ">= 1.0"
   required_providers {
     aws = {
       source  = "hashicorp/aws"
       version = ">= 5.4.0"
     }
+    skopeo2 = {
+      source  = "bsquare-corp/skopeo2"
+      version = "~> 1.1.1"
+    }
     null = {
       source  = "hashicorp/null"
-      version = ">= 3.2.1"
+      version = "~> 3.2.1"
```

Comment:
Do you have a good reason to limit the maximum version here? It is best practice to avoid this in the modules, and put this limit in the root module.

Reply:
no, a mistake from my side; I will fix it